lighthouse-core/gather/gatherers/dobetterweb/document-event-listeners.js

```diff
@@ -115,17 +127,24 @@ class PageLevelEventListeners extends Gatherer {
   }
 
   afterPass(options) {
+    const driver = options.driver;
+
     return this.unlistenForScriptParsedEvents(options.driver)
-      .then(_ => this.collectListeners(LISTENER_LOCATIONS))
-      .then(listeners => {
-        this.artifact = listeners;
-      }).catch(_ => {
-        this.artifact = {
-          usage: -1,
-          debugString: 'Unable to gather passive events listeners usage.'
-        };
-      });
+        .then(_ => driver.sendCommand('DOM.enable'))
```

Comment:
It seems to work without, but I thought I'd need it for `DOM.resolveNode`? FWIW, it looks like the styles gatherer enables it, so maybe that's why it's succeeding? I noticed other places that use DOM.* in driver.js do not call `DOM.enable` before they do their thing.